packages/terra-form-select/src/Select.jsx

```diff
@@ -300,14 +325,7 @@ class Select extends React.Component {
           boundingRef={boundingRef}
         >
           <SelectMenu>
-            <SelectOption
```

Comment:
Can we remove the entire assignment on line 300 and simply do the evaluation as follows?
```jsx
<SelectMenu>
  { !isPlaceholderHidden &&
    <SelectOption
       value={defaultOptionValue}
       display={defaultOptionDisplay}
       isSelected={isDefaultSelected}
       onKeyDown={this.wrapOnKeyDown(defaultOptionValue)}
       onClick={this.wrapOnClick(defaultOptionValue)}
       className={cx('default-option')
    />
  }
  .
  .
  .
```